[1/5] cd /home/utente/Scrivania/univ/iot_as/indiv_proj/indivdual_project/codes/fft/build/esp-idf/esptool_py && /home/utente/.espressif/python_env/idf5.3_py3.10_env/bin/python /home/utente/esp/esp-idf/components/partition_table/check_sizes.py --offset 0x8000 partition --type app /home/utente/Scrivania/univ/iot_as/indiv_proj/indivdual_project/codes/fft/build/partition_table/partition-table.bin /home/utente/Scrivania/univ/iot_as/indiv_proj/indivdual_project/codes/fft/build/fft.bin
fft.bin binary size 0x3fe20 bytes. Smallest app partition is 0x100000 bytes. 0xc01e0 bytes (75%) free.
[2/5] Performing build step for 'bootloader'
[1/1] cd /home/utente/Scrivania/univ/iot_as/indiv_proj/indivdual_project/codes/fft/build/bootloader/esp-idf/esptool_py && /home/utente/.espressif/python_env/idf5.3_py3.10_env/bin/python /home/utente/esp/esp-idf/components/partition_table/check_sizes.py --offset 0x8000 bootloader 0x0 /home/utente/Scrivania/univ/iot_as/indiv_proj/indivdual_project/codes/fft/build/bootloader/bootloader.bin
Bootloader binary size 0x5240 bytes. 0x2dc0 bytes (36%) free.
[2/3] cd /home/utente/esp/esp-idf/components/esptool_py && /usr/bin/cmake -D IDF_PATH=/home/utente/esp/esp-idf -D "SERIAL_TOOL=/home/utente/.espressif/python_env/idf5.3_py3.10_env/bin/python;;/home/utente/esp/esp-idf/components/esptool_py/esptool/esptool.py;--chip;esp32s3" -D "SERIAL_TOOL_ARGS=--before=default_reset;--after=hard_reset;write_flash;@flash_args" -D WORKING_DIRECTORY=/home/utente/Scrivania/univ/iot_as/indiv_proj/indivdual_project/codes/fft/build -P /home/utente/esp/esp-idf/components/esptool_py/run_serial_tool.cmake
esptool.py --chip esp32s3 -p /dev/ttyUSB0 -b 460800 --before=default_reset --after=hard_reset write_flash --flash_mode dio --flash_freq 80m --flash_size 2MB 0x0 bootloader/bootloader.bin 0x10000 fft.bin 0x8000 partition_table/partition-table.bin
esptool.py vv4.8.dev1
Serial port /dev/ttyUSB0
Connecting....

A serial exception error occurred: device reports readiness to read but returned no data (device disconnected or multiple access on port?)
Note: This error originates from pySerial. It is likely not a problem with esptool, but with the hardware connection or drivers.
For troubleshooting steps visit: https://docs.espressif.com/projects/esptool/en/latest/troubleshooting.html
FAILED: CMakeFiles/flash /home/utente/Scrivania/univ/iot_as/indiv_proj/indivdual_project/codes/fft/build/CMakeFiles/flash 
cd /home/utente/esp/esp-idf/components/esptool_py && /usr/bin/cmake -D IDF_PATH=/home/utente/esp/esp-idf -D "SERIAL_TOOL=/home/utente/.espressif/python_env/idf5.3_py3.10_env/bin/python;;/home/utente/esp/esp-idf/components/esptool_py/esptool/esptool.py;--chip;esp32s3" -D "SERIAL_TOOL_ARGS=--before=default_reset;--after=hard_reset;write_flash;@flash_args" -D WORKING_DIRECTORY=/home/utente/Scrivania/univ/iot_as/indiv_proj/indivdual_project/codes/fft/build -P /home/utente/esp/esp-idf/components/esptool_py/run_serial_tool.cmake
ninja: build stopped: subcommand failed.
